docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -163,5 +163,7 @@
       url: /deprecated/Dropdown
     - title: FormGroup
       url: /FormGroup
+    - title: InputField
```

Comment:
General feedback about the docs for these deprecated components:

I think you'll need to move the .md files to `docs/content/deprecated as otherwise they will 404 based on the path below 👇